README.md

```diff
@@ -199,6 +199,7 @@ Configuration reference
   * boot_disk_type - Optional - Guest boot disk type. Default 'thin'.  Available thin, zeroedthick, eagerzeroedthick.
   * boot_disk_size - Optional - Specify boot disk size or grow cloned vm to this size.
   * guestos - Optional - Default will be taken from cloned source.
+  * uefi - Optional - If true, enable efi boot. Warning: changing this setting will force the VM to be recreated. - Default "false" (BIOS boot)
```

Comment:
Please rename this resource to "boot_firmware".  It will more closely match the name in the web gui.  The allowed selection should be "bios", or "efi".  Default="bios"
Why would changing the firmware type for the VM to be recreated????   I can modify it in the console without destroying my vm...

Reply:
fixed.
default value is bios,